src/styles/blocks/_card.scss

```
@@ -0,0 +1,101 @@
+.card {
```

```suggestion
.cards {
```
it includes a lot of cards

---

also, block name shouldn't match element name